### DISCUSSION TITLE
2.6.2 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#1913] [讨论]
· 新增内存优化功能（#1005，@idhaname）
　　在百宝箱中添加了内存优化按钮
　　为自定义主页增加了“内存优化”事件类型
　　使用 --memory 参数启动 PCL 可以静默执行内存优化
· 在选择 OptiFine 与 Fabric 后会自动选择 OptiFabric（#1758，@Chiloven945）
· 清理游戏垃圾后将重启而非关闭 PCL（#1849，@misividkoukou）
· 优化提示文本（ToolTip）的显示
　　提示文本的默认位置均不再根据鼠标位置决定，让显示整体更加美观
　　下拉列表项的提示文本均显示在右侧，以避免遮挡下方的列表项
　　提示文本的背景改为半透明
　　修改、调整了多个提示文本内容
· 更新帮助文档的诸多细节（by @WForst-Breeze、@iPodtouch6557）
· 更新 Mod 中文名数据库
· 翻新了 20 条回声洞（包括把野炊天下第一改成王泪天下第一，时代变了）
· 修复了以下 16 个 Bug
　　特定情况下 Mod 名称分段有误（#1567，@WForst-Breeze）
　　在自动更新进行时尝试手动更新会导致更新失败（#1641，@Chiloven945）
　　特定情况下会尝试为 32 位 Java 分配过多内存（#1687，@Ayx03）
　　愚人节版本在改名后分类有误（#1733，@xtrackc1437）
　　AppData 文件夹未被正常创建（#1746，@520yunchen）
　　下载中的版本会显示在版本列表中（#1748，@iPodtouch6557）
　　罕见情况下报错“检查 Minecraft 窗口失败”（#1836，@AuroralMoss）
　　无法安装 MultiMC Fabric 整合包（#1853，@pangzong2011）
　　资源下载的优先卡片标题有误（#1861，@Chen-Luan）
　　Mixin 相关崩溃分析可能不会给出 Mod 名称（#1875，@MinecraftSTL）
　　启动部分老版本时语言不会默认设为中文（#1877，@Pigmud）
　　默认缓存文件夹无法访问时的报错有误（#1883，@Chormore-EDGE）
　　部分 Mod 下载时的标注名有误（#1893，@Pigmud）
　　在启动新版本后，启动 Minecraft 1.1 ~ 1.5 可能会导致崩溃
　　在新老版本间切换时，语言可能变为英文
　　难以搜索到部分名字含 and、[、] 的 Mod

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.6.2'>2.6.2</a>.</em>

**Comment by Chormore-EDGE** [comment]
打卡

**Comment by WForst-Breeze** [comment]
更新弹窗的版本号是不是有点小问题……还是刻意为之……？🤔
![image](https://github.com/Hex-Dragon/PCL2/assets/110760354/f7fda7d0-e7ff-4dfd-be66-d8719fcd6ab2)

**Reply by LTCatt** [reply]
改了
